.github/workflows/test-all.yml

```diff
@@ -111,13 +168,11 @@ jobs:
       run: |
         cp config/ci.config.json config/project.json
         yarn
-    - name: yarn build
-      run: yarn build:ci:firestore
     - name: Set start timestamp env var
       run: echo "FIREBASE_CI_TEST_START_TIME=$(date +%s)" >> $GITHUB_ENV
     - name: Run unit tests
       run: |
-        xvfb-run yarn test:ci:firestore
+        xvfb-run lerna run --concurrency 4 test:ci --scope '{@firebase/firestore*,firebase-firestore-integration-test}'
```

Comment:
Does reducing concurrency to 1 help with the flakiness?

Reply:
Seems to. Done for both Firestore and Auth!